python/pyspark/sql/column.py

```diff
@@ -329,6 +329,8 @@ def cast(self, dataType):
         [Row(ages=u'2'), Row(ages=u'5')]
         >>> df.select(df.age.cast(StringType()).alias('ages')).collect()
         [Row(ages=u'2'), Row(ages=u'5')]
+        
+	(Note : astype is alias for cast)
```

Comment:
I built and checked the html page. The note appears in both places. Its an alias and the inline documentation is repeated in both places.